test/component/metrics/metrics_actions_test.js

```diff
@@ -226,13 +226,13 @@ describe('Metrics', function() {
 
                                 should.equal(m.services.unknownt.sum.actionEntityUpdate, 1);
                                 should.equal(m.services.unknownt.sum.okActionEntityUpdate, 0);
-                                should.equal(m.services.unknownt.sum.failedActionEntityUpdate, 1);
+                                should.equal(m.services.unknownt.sum.failedActionEntityUpdate, 0);
```

Comment:
This is going to be handled in another PR: https://github.com/telefonicaid/perseo-fe/pull/425

Reply:
In order not disturb @fiqare-emergya-dev (a conflict will happen when #425 gets merged), let's keep as it is now in this PR. I'll fix it in the prelanding branch (hardening/fiqare-perseo-fe-improvements-prelanding)

NTC (just informative)